### DISCUSSION TITLE
Bea Milz - Monitoring quarto-dev repositories: Creating a workflow with GitHub Actions for R users

### DISCUSSION THREAD

**Discussion by giscus** [beatrizmilz/blog-en#23] [General]
# Bea Milz - Monitoring quarto-dev repositories: Creating a workflow with GitHub Actions for R users

Welcome to my blog! Here I write content about R and data science.

https://beamilz.com/posts/series-gha/2022-series-gha-2-creating-your-first-action/en/

**Comment by ghost** [comment]
Thanks for your great post, I always wanted to automate processes with cron but I see that yours is more complete.
